src/Trixi.jl

```diff
@@ -159,7 +159,7 @@ export LaplaceDiffusion1D, LaplaceDiffusion2D, LaplaceDiffusion3D,
 
 export GradientVariablesPrimitive, GradientVariablesEntropy
 
-export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hllc, flux_hlle,
+export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hllc,
```

Comment:
Can we please get `flux_hlle` back (and `export` it) by adding something like 

https://github.com/trixi-framework/Trixi.jl/blob/61c33b0af6a7a49ed11258e8f230b471b05c6ed8/src/equations/numerical_fluxes.jl#L305

with an appropriate docstring linking to `min_max_speed_einfeldt`?